std/array.d

```diff
@@ -442,18 +451,97 @@ if (isInputRange!Range)
     return aa;
 }
 
+/// ditto
+auto assocArray(Keys, Values)(Keys keys, Values values)
+if (isInputRange!Values && isInputRange!Keys)
+{
+    static if (isDynamicArray!Keys && isDynamicArray!Values
+        && !isNarrowString!Keys && !isNarrowString!Values)
```

Comment:
Added `&& !isNarrowString!Keys && !isNarrowString!Values` for consistency with other Phobos functions that treat strings as ranges.